### ISSUE TITLE
Input[type="search"] has incorrect `box-sizing` property (a.k.a Update normalize). 

### ISSUE THREAD

**Issue by chemoish** [zurb/foundation-sites#8588]
**How can we reproduce this bug?**
1. Create `<input type="search" />`
2. See that `box-sizing: content-box` is set
3. See that `normalize.css` is out of date

**What did you expect to happen?**

Expected `normalize.css` **NOT** to be embedded in foundation (Seems like its hard to upgrade—maybe there is a better way of handling this [see below for context]?).

7 days after release of `3.0.3`, https://github.com/necolas/normalize.css/commit/fe567632afcf8afcf17925ab2a507ed87504cce8 was introduced. Also, normalize is currently at `4.1.0` (https://github.com/necolas/normalize.css/releases/tag/4.1.0).

TLDR, just recommendation, overall extreme limited context of this repo.

**What happened instead?**

See that `@include foundation-global-styles`, added in https://github.com/zurb/foundation-sites/issues/7159, is on version `v3.0.3`.

**Test case:**

http://codepen.io/chemoish/pen/NNYWEw

**Comment by andycochran** [comment]
I'm all for making this change: 

<img width="294" alt="screen shot 2016-04-21 at 4 56 39 pm" src="https://cloud.githubusercontent.com/assets/409279/14724056/1228891a-07e2-11e6-9a66-8d2158508fea.png">

But I don't know the reasons why Normalize is hardcoded.

**Comment by Lynda333** [comment]
+1 for updating Normalize.

**Comment by rgalus** [comment]
+1 for updating Normalize

**Comment by DanielRuf** [comment]
normalize-scss is a fork of normalize and does not provide the latest changes from normalize. Imo we should switch to a pure normalize implementation.

**Comment by DanielRuf** [comment]
Closing as https://github.com/zurb/foundation-sites/pull/11086 is merged now.

Feel free to reopen if the error still exists.
